# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523585598 25200
#      Thu Apr 12 19:13:18 2018 -0700
# Branch stable
# Node ID fd7f2108c991d99b52a0746b309163aac5ecdd5f
# Parent  e1024058833c614ac9bee2e3e5c57ad55a9e1d48
NEWS: Reword deletion of --force-gui option.

* NEWS: Reword deletion of --force-gui option.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -16,21 +16,22 @@ Summary of important user-visible change
       BLAS  LAPACK  QRUPDATE  SuiteSparse  ARPACK
 
     Additionally, the following libraries use "int" internally, so
     maximum problem sizes are always limited:
 
       glpk  Qhull
 
  ** The octave command no longer starts the GUI by default.  Most users
-    starting Octave from command line were expecting the command line
-    interface and Desktop launchers required the `--force-gui' option
-    anyway.  With this change, desktop launchers should change to use
-    the new option `--gui'.  The previous `--force-gui' will continue to
-    work and maps to `--gui' but will be removed in Octave 6.
+    starting Octave from a shell were expecting the command line
+    interface, and desktop launchers already required the `--force-gui'
+    option.  With this change, desktop launchers should be modified to
+    use the new option `--gui'.  The previous `--force-gui' option will
+    continue to work, and maps to `--gui', but it will be removed in
+    Octave 6.
 
  ** A known bug in Qt (https://bugreports.qt.io/browse/QTBUG-55357) is
     addressed by limiting GUI sub-panel relocation capabilities for Qt
     versions in the range >= 5.6.1 and < 5.7.1.  However, this may not
     thoroughly avoid issues on all platforms.
 
  ** A new container data type--containers.Map--is available.  Map is a
     key/value storage container (a.k.a, a hash) that efficiently allows
